changelog/extensions.feature.md

```diff
@@ -0,0 +1 @@
+Add pluggy to provide ability to create sanic extensions in rasa-sdk
```

Comment:
Please replace extensions in the file name with `970` the PR id, towncrier might not be able to generate the correct PR link for this changelog entry otherwise.